packages/o365/manifest.yml

```
@@ -29,7 +29,10 @@ policy_templates:
     description: Collect logs from Office 365
     inputs:
       - type: o365audit
-        title: "Collect Office 365 audit logs"
-        description: "Collect audit logs from Office 365 via the Management Activity API"
+        title: "Collect Office 365 audit logs - Deprecated. Please use the CEL input instead."
```

```suggestion
        title: "Collect Office 365 audit logs - Deprecated. Please disable this and use the CEL input instead."
```

I was thinking the message somewhere should tell them to disable it explicitly.

---

Updated the manifest as suggested.